### PR TITLE
BAM-17619: Updated clickmate endpoint

### PR DESCRIPTION
**Pull request by ivorytoast**

### CODE REVIEW

File: docs/clickmate.rst [hunk 1]
```diff
@@ -40,7 +40,7 @@ The legacy url domain of shop-links.co is still supported.
 
 ::
 
-   https://shop-links.co/link/?url=https%3A%2F%2Fdermstore.com.example%2Fshop%2Fproduct1234&publisher_slug=myacct&article_name=my-story
+   https://shop-links.co/link/?url=https%3A%2F%2Fdermstore.com.example%2Fshop%2Fproduct1234&publisher_slug=myacct&article_name=my-story&article_url=
```
Comment (TeresaNgo-gh) [inline docs/clickmate.rst:43]: Is there more to the article_url param example?

Comment (ivorytoast) [reply docs/clickmate.rst:43]: This is just a "template" -- it is meant to say, "add in your own article_url"

Comment (TeresaNgo-gh) [reply docs/clickmate.rst:43]: Yes, but could it be an example thing (like how article_name=my-story and publisher_slug=myacct)? 
Also @cheukhangko I still don't see the howl.me changes in the API docs? https://docs.narrativ.com/en/stable/clickmate.html

Comment (ivorytoast) [reply docs/clickmate.rst]: Good point, I added "https://www.mywebpage.com" (encoded)

**Review by TeresaNgo-gh — APPROVED** [review]

**Pull request merged**